src/lib/storage/value_segment.hpp

```diff
@@ -13,7 +13,12 @@ namespace opossum {
 template <typename T>
 class ValueSegment : public BaseSegment {
  public:
-  // return the value at a certain position. If you want to write efficient operators, back off!
+
+    ValueSegment() {
+        _value_segment = std::vector<T>();
```

Comment:
This is not necessary because `std::vector` gets implicitly initialized like this already in l. 34